src/style.css

```diff
@@ -0,0 +1,180 @@
+@font-face {
+    font-family: Harry;
+    src: url("images/Parry\ Hotter.ttf");
+  }
+
+*{
+    margin:0;
+    padding:0;
+    box-sizing: border-box;
+    font-family: "Harry";
+    color:rgb(245, 248, 71);
+}
+html{
+    font-size:0.7vw;
+}
+a{
+    color:#EFF24E;
+    text-decoration:none;
+    font-size:2rem;
+}
+body {
+    background-image: url("images/f9d7af29e8b87e7bdbc81609503036d1.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    background-position: bottom;
+}
+main{
+    position:relative;
+    height:100vh;
+}
+@media only screen and (min-width: 600px) {
+    main{
+    height:190vh;
+}
+}
+.main_feather, .main_book_shadow, .h_hogwart{
+    width:7vw;
+}
+
+nav{
+    position:fixed;
+    height:25%;
+    width:100%;
+    display:flex;
+    z-index:1;
+}
+.main_button{
+    position:absolute;
+    top:30%;
+    left:5%;
+    width:20%;
+
+}
+.main_button img:hover{
+        filter: drop-shadow(2px 2px 0 rgb(231, 108, 7))
+        drop-shadow(-2px 2px 0 rgb(223, 122, 6))
+        drop-shadow(2px -2px 0 white)
+        drop-shadow(-2px -2px 0 white);
+}
+.main_button h3{
+    position:absolute;
+    display:inline-block; 
+    font-size:4rem;
+    left:50%;
+    text-shadow:2px 2px 3px #FA6900;
+}
+
+.teach_spells{
+    position:absolute;
+    top:30%;
+    right:5%;
+    display:flex;
+    width:30%;
+    align-items:center;
+    justify-content:space-around;
+}
+.teach_spells img:hover{
```

Comment:
Jeśli napisy 'Teachers' i 'Spells' też są linkami to wydaję mi się, że dobrze by było gdyby po najechaniu na nie też pojawiał się cień na obrazkach. I w sumie można by to ujednolicić, że skoro tutaj napisy są też linkami to może dla domów zrobić to samo?